src/components/Error.jsx

```diff
@@ -96,6 +96,7 @@ const addLogAttachmentCommentToReportURL = (reportURL) => {
 const addOSPrettyNameCommentToReportURL = (reportURL, prettyName) => {
     const newUrl = new URL(reportURL);
     const comment = newUrl.searchParams.get("comment") || "";
+    // eslint-disable-next-line quotes
     newUrl.searchParams.set("comment", "\n\n" + cockpit.format(_('Issue was detected on "$0"'), prettyName) + comment);
```

Comment:
I disabled eslint for the next line because I personally find the escaped quotes (the fix) distracting. for reference, it would look like this

`..."Issue was detected on \"$0\""`

But if the team prefers it escaped, I can remove it. No problem :)